src/app/CommandHandler.h

```diff
@@ -374,13 +347,12 @@ class CommandHandler
      * Most applications are likely to use `AddResponseData` as a more convenient
      * one-call that auto-sets command ID and creates the underlying encoders.
      */
-    CHIP_ERROR AddResponseDataViaEncoder(const ConcreteCommandPath & aRequestCommandPath, CommandId commandId,
-                                         EncoderToTLV & encoder)
+    CHIP_ERROR AddResponseData(const ConcreteCommandPath & aRequestCommandPath, CommandId commandId,
+                               DataModel::EncoderToTLV & encoder)
```

Comment:
```suggestion
                               DataModel::EncoderToTLV & aEncoder)
```
But note comments below about EncoderToTLV naming.